app/styles/front.scss

```diff
@@ -227,13 +227,16 @@ $autre-lighter: #cce4e7;
 
 
 /* Should be removed once all buttons have been converted to native html input fields. */
-.btn-default {
+.btn-default, .btn {
```

Comment:
Comment is outadated, since we moved back from the native html toswar angular-style `ng-class` in 1d8246f9e0ae5350bea29e113df267f10d7dea6a.

Let's remove it